.github/workflows/linux_builds.yml

```diff
@@ -126,17 +126,47 @@ jobs:
           scons --version
 
       # We should always be explicit with our flags usage here since it's gonna be sure to always set those flags
+      # [Workaround] SwiftShader have problems with tesselation, so we comment this for now
       - name: Compilation
         env:
           SCONS_CACHE: ${{github.workspace}}/.scons_cache/
         run: |
-          scons tools=yes tests=yes target=debug module_mono_enabled=yes mono_glue=no use_asan=yes use_ubsan=yes
+          sed -i "s|ERR_FAIL_COND_V(p_rasterization_state.patch_control_points|//ERR_FAIL_COND_V(p_rasterization_state.patch_control_points|" drivers/vulkan/rendering_device_vulkan.cpp
+          scons tools=yes tests=yes target=debug use_asan=yes use_ubsan=yes
           ls -l bin/
 
       # Execute unit tests for the editor
       - name: Unit Tests
         run: |
-          ./bin/godot.linuxbsd.tools.64s.mono --test
+          ./bin/godot.linuxbsd.tools.64s --test
+
+      # Download, unzip and setup SwiftShader library
+      - name: Download SwiftShader
+        run: |
+          wget https://github.com/qarmin/gtk_library_store/releases/download/3.24.0/swiftshader.zip
```

Comment:
It would be good to document what version of SwiftShader this is / how it was built. I just spent 5 minutes going through the upstream repo trying to find any reference to a `3.24.0` release...

Reply:
I added commit number to comment.
Compilation instructions are in https://github.com/godotengine/godot/issues/38428#issuecomment-808407836 and https://github.com/godotengine/godot/issues/38428#issuecomment-808712091

SwiftShader is currently developing quite slowly, so I think an update will only be necessary in no less than 3 months.